aggregators/aggregator.go

```diff
@@ -447,8 +447,19 @@ func (a *Aggregator) harvest(
 		// Check if the given aggregation interval needs to be harvested now
 		if end.Truncate(ivl).Equal(end) {
 			start := end.Add(-ivl).Add(-a.cfg.Lookback)
+			intervalMetrics := cachedEventsStats[ivl]
+			if intervalMetrics == nil {
```

Comment:
Can you add a comment here as to when this will happen? The issue description of https://github.com/elastic/apm-aggregation-service/issues/410 has the context.